sds_data_manager/lambda_code/SDSCode/batch_starter.py

```diff
@@ -311,48 +323,101 @@ def prepare_data(instrument, data_level, start_date, version, upstream_dependenc
     return prepared_data
 
 
-def send_lambda_put_event(command_parameters):
+def check_duplicate_job(
```

Comment:
Could we change this to an explicit name about what we are doing?
```suggestion
def is_job_in_status_table(
```